src/components/Person/Person.jsx

```diff
@@ -1,30 +1,19 @@
 export const Person = ({ person }) => {
   const { name, age, sex, isMarried, partnerName } = person;
-  let partnerSex;
 
-  if (sex === 'm') {
-    partnerSex = 'wife';
-  } else {
-    partnerSex = 'husband';
-  }
-
-  let partnerInfo;
-
-  if (isMarried) {
-    partnerInfo = (
-      <p className="Person__partner">
-        {partnerName} is my {partnerSex}
-      </p>
-    );
-  } else {
-    partnerInfo = <p className="Person__partner">I am not married</p>;
-  }
+  const partnerSex = sex === 'm' ? 'wife' : 'husband';
 
   return (
     <section className="Person">
       <h2 className="Person__name">My name is {name}</h2>
       {age && <p className="Person__age">I am {age}</p>}
-      {partnerInfo}
+      {isMarried ? (
+        <p className="Person__partner">
+          {partnerName} is my {partnerSex}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
Format ternary operator operands correctly [[checklist]](https://github.com/mate-academy/react_person/blob/master/checklist.md)

Reply:
I changed, but prittier marked everything in red and changed back after commit to the previous.

Reply:
sorry I didn't realize that prettier has different format settings ^^"